src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const invertRobot = {};
+
+  for (const key in robot) {
+    const property = robot[key];
+
+    if (invertRobot.hasOwnProperty(property)) {
+      return null;
+    } else {
+      invertRobot[property] = key;
+    }
```

Comment:
you can remove else block
```suggestion
    if (!invertRobot.hasOwnProperty(property)) {
      invertRobot[property] = key;
    }
```